### PR TITLE
Updated whats next content

### PR DESCRIPTION
**Pull request by arthurHarding**

### CODE REVIEW

File: CooperationHullMainSite/wwwroot/content/homepageContent.json [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
+      "eventLink": "https://www.instagram.com/p/C8uHqcztOz7/"
     },
     {
-      "title": "HU4 Neighbourhood Assembly",
-      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eget dui tortor. Fusce blandit elit sit amet.",
-      "date": "2024-02-01T00:00:00",
-      "imagesName": "pattern 2.jpeg",
-      "eventLink": "https://www.instagram.com/p/C3vqGwUo431/"
+      "title": "Facilitation Training",
+      "description": "Facilitators are the glue that hold our assemblies together, and we need you to be one! Stick around after for a social. 6 - 7pm at The Three Crowns on Vistoria Dock.",
```
Comment (Fireflower2012) [inline CooperationHullMainSite/wwwroot/content/homepageContent.json:13]:
```suggestion
      "description": "Facilitators are the glue that hold our assemblies together, and we need you to be one! Stick around after for a social. 6 - 7pm at The Three Crowns on Victoria Dock.",
```

File: CooperationHullMainSite/wwwroot/content/homepageContent.json [hunk 2]
```diff
@@ -2,25 +2,25 @@
...
+      "eventLink": "https://www.instagram.com/p/C7ybjUYpUuO/"
     },
     {
-      "title": "HU5 Neighbourhood Assembly",
-      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Maecenas eget dui tortor. Fusce blandit elit sit amet.",
-      "date": "2024-03-01T00:00:00",
-      "imagesName": "pattern 1.jpeg",
-      "eventLink": "https://www.instagram.com/p/C3IOg8WNXvI/?img_index=1"
+      "title": "The Peoples Party II",
+      "description": "If I cant dance, its not my revolution. Come for the music, stay to be a part of a historic movement for change. 6pm til late, venue TBC",
```
Comment (Fireflower2012) [inline CooperationHullMainSite/wwwroot/content/homepageContent.json:20]:
```suggestion
      "description": "If I can\'t dance, it\'s not my revolution. Come for the music, stay to be a part of a historic movement for change. 6pm til late, venue TBC",
```

**Review by Fireflower2012 — CHANGES_REQUESTED** [review]
Couple of typos

File: CooperationHullMainSite/wwwroot/content/homepageContent.json (hunk 2 above)
Comment (arthurHarding) [reply CooperationHullMainSite/wwwroot/content/homepageContent.json:20]: The backslash escapes don't work for me. Think I'll leave figuring out apostrophes for a future time

**Review by lewis-conroy — APPROVED** [review]

**Pull request merged**